addon/-private/system/model/internal-model.js

```diff
@@ -543,7 +543,7 @@ InternalModel.prototype = {
     came back from the server, except the user obtained them out of band and is informing
     the store of their existence. The most common use case is for supporting client side
     nested URLs, such as `/posts/1/comments/2` so the user can do
-    `store.find('comment', 2, {post:1})` without having to fetch the post.
+    `store.findRecord('comment', 2, {post:1})` without having to fetch the post.
```

Comment:
I think this should be `store.findRecord('comment', 2, { preload: { post: 1 } })`